gatsby-config.js

```diff
@@ -44,7 +44,16 @@ module.exports = {
       access_key: 'xX1gtb2RcnLbIYkHAcB6IaTRr4ZfN-p16ofcyUebeko=', // This is placeholder. Not private.
       inline: true, // this renders the results on the same domain. Otherwise, it will render the results in the search.gov domain
     },
+
+    // DAP configuration
     dapAgency: 'GSA',
```

Comment:
if a user fails to change the default dapAgency will that mess with the stats for GSA? if so, should we default to something else

Reply:
or will the site stats exist for the site but sorted under GSA in DAP?